Marlin/Configuration.h

```diff
@@ -1538,8 +1538,45 @@
   #define RGB_LED_G_PIN 43
   #define RGB_LED_B_PIN 35
   #define RGB_LED_W_PIN -1
+  
```

Comment:
Search for regex `[ \t]+$` and replace with nothing to clean up excess whitespace.